src/Stepper/index.js

```diff
@@ -0,0 +1,76 @@
+// @flow srict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Text from '../Text';
+import Icon from '../Icon';
+import StepperButton from './StepperButton';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +onIncrement: () => void,
+  +onDecrement: () => void,
+  +number: number,
+  +showNumber?: boolean,
+  +numberStyle?: StylePropType,
+  +min?: number,
+  +max?: number,
+  +disableDecrement?: boolean,
+  +disableIncrement?: boolean,
+|};
+
+export default function Stepper({
+  onIncrement,
+  onDecrement,
+  number,
+  showNumber = true,
+  numberStyle,
+  min,
+  max,
+  disableDecrement: originalDisableDecrement,
+  disableIncrement: originalDisableIncrement,
+}: Props) {
+  const belowMin = min && number <= min;
```

Comment:
Would not `number <= min` be sufficient since
1 <= undefined -> `false`

Reply:
I would get the following Flow error:
```
Cannot compare number [1] to undefined [2]. (References: [1] [2])
```